### PR TITLE
Enable dynamic input switching

### PR DESCRIPTION
**Pull request by utzcoz**
The `vendor_maruos` part to enable dynamic input switching. For the whose view, please visit the issue [Enable dynamic input switching](https://github.com/maruos/maruos/issues/97).

### CODE REVIEW

File: perspective/LXCContainerManager.cpp [hunk 1]
```diff
@@ -116,6 +118,24 @@ static bool containerIsRunning(const char* id) {
...
+static bool containerEnableInput(const char* id, const bool enable) {
+    struct lxc_container *c = initContainer(id);
+    if (!c) {
+        ALOGE("failed to enable input of container, can't init container");
+        return false;
+    }
+
+    const char *program = enable ? "/etc/maruos/enable-input" : "/etc/maruos/disable-input";
+    lxc_attach_options_t attach_options = LXC_ATTACH_OPTIONS_DEFAULT;
+    // The lxc_attach_command_t.argv in attach_options.h needs including
```
Comment (pdsouza) [inline perspective/LXCContainerManager.cpp:132]: Nice work figuring out how to use `lxc-attach`!

File: device-maru.mk [hunk 2]
```diff
@@ -38,6 +34,7 @@ PRODUCT_PACKAGES += \
 	liblxc \
 	lxc-start \
 	lxc-stop \
+	lxc-attach \
```
Comment (pdsouza) [inline device-maru.mk:37]: Yes, this is good for debugging - thanks.

**Review by pdsouza — APPROVED** [review]
Great work @utzcoz! Your PR to do dynamic input switching has touched basically all the core parts of Maru!

**Comment by utzcoz** [comment]
@pdsouza  I have squashed all commits with two main commits for cleaning up.

**Comment by utzcoz** [comment]
@pdsouza I have added a new patch to drop `PID` namesapce when running lxc-attach.

**Comment by pdsouza** [comment]
@utzcoz Merged - thanks for your patience!

**Pull request merged**